app/models/contact.rb

```diff
@@ -0,0 +1,5 @@
+class Contact < ActiveRecord::Base
+
+  
+
+end
```

Comment:
should delete this if its not going to be used
`rails d model contact`
if I do this, to be safe, I always reset or drop my db and migrate again